src/pydata_sphinx_theme/assets/styles/_markdown.scss

```diff
@@ -1,16 +0,0 @@
-// Rules that are unique to MyST markdown parsing / outputs
-
-th,
-td {
-  &.text-align\:left {
-    text-align: left;
-  }
-
-  &.text-align\:right {
-    text-align: right;
-  }
-
-  &.text-align\:center {
-    text-align: center;
-  }
-}
```

Comment:
This is actually removed (and not migrated). Any reasons for the removal?

Reply:
oops - accidental, this is the "missing" tables.scss file that @12rambau noted...I've added it back in